core/api/serializers/project.py

```diff
@@ -467,16 +470,33 @@ def get_metaproject_category(self, obj):
             return None
         return obj.meta_project.type
 
+    def create(self, validated_data):
```

Comment:
I don' think create method is ever called as POST is handled by ProjectDetailSerializer and, even though it inherits ProjectListSerializer, the create method there doesn't call super.

For update, I see that ProjectDetailSerializer.update calls super, but I think it would be better to keep all the logic in the same place (in ProjectDetailSerializer.update)

Reply:
Moved to `ProjectDetailSerializer`.